bigtable/client/instance_admin.h

```diff
@@ -112,6 +112,18 @@ class InstanceAdmin {
    */
   std::vector<google::bigtable::admin::v2::Cluster> ListClusters();
 
+  /**
+   * Deletes the specified cluster of an instance in the project.
+   * @param instance_id the id of the instance in the project
+   * @param cluster_id the id of the cluster in the project that needs to be
+   *   deleted
+   *
+   *  **Example**
```

Comment:
Thanks for writing the sample in the first PR!  We switched these to `\par Example`.

Reply:
fixed